Doc/library/sqlite3.rst

```diff
@@ -2614,6 +2583,65 @@ With some adjustments, the above recipe can be adapted to use a
 instead of a :class:`~collections.namedtuple`.
 
 
+.. _sqlite3-howto-text-factory:
+
+How to create and use text factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` adapts SQLite values with the ``TEXT`` data type
+using :class:`str`.
+This works well for correctly encoded UTF-8 text, but it will fail for invalid
+Unicode sequences and other encodings.
+To work around this, you can use a custom :attr:`~Connection.text_factory`.
+
+Because of SQLites flexible typing, it is not uncommon to encounter table
+columns with the ``TEXT`` data type, containing arbitrary data.
+Let's create a test database with an invalid Unicode sequence:
```

Comment:
Maybe make clear that this isn't necessarily a _recommendation_, just a demonstration:

```suggestion
To demonstrate, let's create a test database with an invalid Unicode sequence:
```